pom.xml

```diff
@@ -11,7 +11,7 @@
 	<groupId>io.jenkins.services.distribution-service</groupId>
 	<artifactId>custom-distribution-service</artifactId>
 	<version>1.0.0</version>
-	<name>Custom-Jenkins-Distribution-Service</name>
+	<name>custom-distribution-service</name>
```

Comment:
We can just let it be `Custom Jenkins Distribution Service`. For example https://github.com/jenkinsci/jenkins/blob/c8bd0f0949b0a4467c52c594a3f7df973ee8e358/pom.xml#L40

Reply:
Do we need the word Jenkins in there ? since it is already in the JenkinsCI organization could it just be Custom Distribution Service ?

Reply:
It's no harm if we leave it. I suggest that we keeping it.

Reply:
Done :+1: